content/src/content/docs/learn/tutorials/hooking-up-a-database.mdx

```diff
@@ -0,0 +1,466 @@
+---
+type: Tutorial
+title: Hooking up a database
+tags:
+  - some
+  - tags
+sidebar:
+  order: 3
+---
+
+import { Aside } from "@astrojs/starlight/components"
+
+The next endpoint we're going to write is one that lists all of the users in a
+database. For this tutorial I'm going to use Postgres running in Docker as our
+database. If you don't have Docker installed, I recommend following the
+[official getting started guide](https://www.docker.com/get-started/).
+
+When you're ready, you can start a Postgres container with:
+
+```shell showLineNumbers=false
+docker run -d -e POSTGRES_HOST_AUTH_METHOD=trust -p 5432:5432 postgres:17
+```
+
+<Aside type="note" title="I can't use Docker!">
+If you aren't able to run Docker locally, for example because your work's laptop
+policy doesn't allow it, an alternative is to use a hosted Postgres service.
+For example, [Neon](https://neondb.com/) offers a free tier that should be
+more than enough for this tutorial. They also offer a web-based UI for creating
+tables and rows.
```

Comment:
While I understand wanting to point users in the right direction, I'm not sure we should direct them to a _specific_ hosted service.